AnkiDroid/src/main/java/com/ichi2/libanki/Utils.java

```diff
@@ -460,7 +460,8 @@ public static String ids2str(JSONArray ids) {
 
 
     /** LIBANKI: not in libanki */
-    public static long[] arrayList2array(List<Long> list) {
+    /** Transform a collection of Long into an array of Long */
+    public static long[] collection2Array(java.util.Collection<Long> list) {
```

Comment:
doesn't need to be fully qualified?

```suggestion
    public static long[] collection2Array(Collection<Long> list) {
```

Reply:
We need it, because we already have Anki Collecrion
Com.ichi2.libanki.Collection

Reply:
`Collection` is typically the libAnki class, it's a common issue